docs/content/en/tasks/backup-restore/_index.md

```diff
@@ -154,58 +149,199 @@ status:
-Once the K8ssandra Operator detects on the `CassandraRestore` object creation, it will control the shutdown of all Cassandra pods, and the `medusa-restore` container will perform the actual data restore upon pod restart.
+The `spec.backup` value should match the `MedusaBackup` `metadata.name` value.  
+Once the K8ssandra Operator detects on the `MedusaRestoreJob` object creation, it will orchestrate the shutdown of all Cassandra pods, and the `medusa-restore` container will perform the actual data restore upon pod restart.
 
 ### Checking Restore Completion
 
-To monitor the restore completion, check if the `finishTime` value isn't empty in the `CassandraRestore` object status. Example:
+To monitor the restore completion, check if the `finishTime` value isn't empty in the `MedusaRestoreJob` object status. Example:
 
 ```sh
 % kubectl get cassandrarestore/restore-backup1 -o yaml
 
 apiVersion: medusa.k8ssandra.io/v1alpha1
-kind: CassandraRestore
+kind: MedusaRestoreJob
 metadata:
   name: restore-backup1
 spec:
   backup: medusa-backup1
-  cassandraDatacenter:
-    clusterName: demo
-    name: dc1
-  inPlace: true
-  shutdown: true
+  cassandraDatacenter: dc1
 status:
   datacenterStopped: "2022-01-06T16:45:09Z"
   finishTime: "2022-01-06T16:48:23Z"
   restoreKey: ec5b35c1-f2fe-4465-a74f-e29aa1d467ff
+  restorePrepared: true
   startTime: "2022-01-06T16:44:53Z"
 ```
 
+## Synchronizing MedusaBackup objects with a Medusa storage backend (S3, GCS, etc.)
+
+In order to restore a backup taken on a different cluster, a synchronization task must be executed to create the corresponding `MedusaBackup` objects locally.  
+This can be achieved by creating a `MedusaTask` custom resource in the namespace where K8ssandra was deployed, with a `sync` operation:
+
+```yaml
+apiVersion: medusa.k8ssandra.io/v1alpha1
+kind: MedusaTask
+metadata:
+  name: sync-backups-1
+  namespace: k8ssandra-operator
+spec:
+  cassandraDatacenter: dc1
+  operation: sync
+```
+
+Reconciliation will be triggered by the `MedusaTask` object creation, executing the following operations:
+
+- Backups will be listed in the remote storage system
+- Backups missing locally will be created
+- Backups missing in the remote storage system will be deleted locally
+
+Upon completion, the `MedusaTask` object status will be updated with the finish time and the name of the pod which was used to communicate with the storage backend:
+
+```yaml
+...
+status:
+  finishTime: '2022-07-26T08:15:55Z'
+  finished:
+    - podName: demo-dc2-default-sts-0
+  startTime: '2022-07-26T08:15:54Z'
+...
+```
+
+## Purging backups
+
+Medusa has two settings to control the retention of backups: `max_backup_age` and `max_backup_count`.
+These settings are used by the `medusa purge` operation to determine which backups to delete.
+In order to trigger a purge, a `MedusaTask` custom resource should be created in the namespace where K8ssandra was deployed, with the `purge` operation:
```

Comment:
Shouldn't this be the namespace/cluster in which the CassandraDatacenter exists?

Reply:
Well, for obvious reasons the same error is everywhere 😅